packages/eslint-config-recommended/package.json

```diff
@@ -37,19 +37,19 @@
   },
   "dependencies": {
     "@shiftcode/eslint-plugin-rules": "^2.0.1",
-    "@typescript-eslint/eslint-plugin": "^7.16.0",
-    "@typescript-eslint/parser": "^7.16.0",
+    "@typescript-eslint/eslint-plugin": "^8.16.0",
+    "@typescript-eslint/parser": "^8.16.0",
     "eslint": "^8.56.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.28.1",
     "eslint-plugin-unused-imports": "^3.2.0"
   },
   "peerDependencies": {
-    "@angular-eslint/builder": "^18.0.0",
-    "@angular-eslint/eslint-plugin": "^18.0.0",
-    "@angular-eslint/eslint-plugin-template": "^18.0.0",
-    "@angular-eslint/schematics": "^18.0.0",
-    "@angular-eslint/template-parser": "^18.0.0"
+    "@angular-eslint/builder": "^19.0.2",
```

Comment:
if there is no explicit reason on why `19.0.0` & `19.0.1` are not allowed, I would just use `^19.0.0` as peerDep definition.

Reply:
@michaelwittwer I based my commit message on your analog 
<img width="957" alt="Screenshot 2024-12-11 at 11 37 05" src="https://github.com/user-attachments/assets/51a94e23-4c8e-4135-8bd8-d1279d0ed375">